### PR TITLE
Change UV transformer cable to bedrockium so you can actually make it in UV

### PR DESCRIPTION
**Pull request by Prometheus0000**
See https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/7611

Change anycopper to copper for LV one to reduce confusion, and because I doubt anyone would use annealed

Change size from x2 or x4 to x1, because it doesn't make sense for them to be larger, they're not high-amp transformers, they send the same amount of amps as the earlier tiers. The cables already cost more

### CODE REVIEW

**Comment by botn365** [comment]
people should still be able to use anneald copper if they want

**Comment by Prometheus0000** [comment]
Is there really a use case for that though? And personally I found it confusing, like maybe using annealed would make it better somehow, when I first made one.

File: src/main/java/gregtech/loaders/preload/GT_Loader_MetaTileEntities.java [hunk 1]
```diff
@@ -177,14 +177,14 @@ private static void run1() {
         ItemList.Transformer_MAX_UV.set(new GT_MetaTileEntity_Transformer(28, "transformer.tier.08", "Ultimate Transformer", 8, "UHV -> UV (Use Soft Mallet to invert)").getStackForm(1L));
 
         GT_ModHandler.addCraftingRecipe(ItemList.Transformer_LV_ULV.get(1L), bitsd, new Object[]{" BB", "CM ", " BB", 'M', ItemList.Hull_ULV, 'C', OrePrefixes.cableGt01.get(Materials.Tin), 'B', OrePrefixes.cableGt01.get(Materials.Lead)});
-        GT_ModHandler.addCraftingRecipe(ItemList.Transformer_MV_LV.get(1L), bitsd, new Object[]{" BB", "CM ", " BB", 'M', ItemList.Hull_LV, 'C', OrePrefixes.cableGt01.get(Materials.AnyCopper), 'B', OrePrefixes.cableGt01.get(Materials.Tin)});
-        GT_ModHandler.addCraftingRecipe(ItemList.Transformer_HV_MV.get(1L), bitsd, new Object[]{"KBB", "CM ", "KBB", 'M', ItemList.Hull_MV, 'C', OrePrefixes.cableGt01.get(Materials.Gold), 'B', OrePrefixes.cableGt01.get(Materials.AnyCopper),'K',ItemList.Circuit_Parts_Coil});
+        GT_ModHandler.addCraftingRecipe(ItemList.Transformer_MV_LV.get(1L), bitsd, new Object[]{" BB", "CM ", " BB", 'M', ItemList.Hull_LV, 'C', OrePrefixes.cableGt01.get(Materials.Copper), 'B', OrePrefixes.cableGt01.get(Materials.Tin)});
```
Comment (Dream-Master) [inline src/main/java/gregtech/loaders/preload/GT_Loader_MetaTileEntities.java:180]: please change it back.

**Review by Dream-Master — CHANGES_REQUESTED** [review]

File: src/main/java/gregtech/loaders/preload/GT_Loader_MetaTileEntities.java (hunk 1 above)
Comment (Prometheus0000) [reply src/main/java/gregtech/loaders/preload/GT_Loader_MetaTileEntities.java]: Changed back, but I stand by my opinion that it's confusing and unnecessary.

**Review by Dream-Master — APPROVED** [review]

**Comment by johnch18** [comment]
Wish this was in a month ago :'( had to make UHV supes for my transformers.

**Comment by Prometheus0000** [comment]
It's always faster to do it yourself. Also, it's not in a release or anything yet.

**Comment by Dream-Master** [comment]
will be in 2.1.0.7dev

**Pull request merged**